### ISSUE TITLE
Stadia Home as landing page

### ISSUE THREAD

**Issue by tychoregter** [liberodark/stadia-app#3]
I think it would be great if the page you land on when opening the app would be the Stadia Home page instead of the Google account selection/login page.

**Comment by STiAT** [comment]
Yeah, annoyed me too, especially because you do not get to Stadia directly when launching. Just switch:
"https://accounts.google.com/SignOutOptions?continue=https%3A%2F%2Fstadia.google.com%2F"
for
"https://stadia.com"

in the .sh and run it.

**Comment by Hitmn1** [comment]
Can anyone compile a MAC-Version, when the App is starting, it begins directly on the Stadia screen and not with the Google Account Page. Thanx.
